frontend/mono-ui/web/dss-dashboard/src/utils/commons.js

```
@@ -28,9 +28,19 @@ const removeSpaceInLocalisationKey = (key = "") => {
 }
 
 export const getTenantId = () => {
-    let tenant=sessionStorage.getItem('Digit.Employee.tenantId');
-    return `${JSON.parse(tenant)?.value||localStorage.getItem('tenant-id')}`;
-}
+    const tenantRaw = sessionStorage.getItem('Digit.Employee.tenantId');
+    let tenantValue = null;
+  
+    try {
+      const parsed = JSON.parse(tenantRaw);
+      tenantValue = parsed && parsed.value ? parsed.value : null;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)